src/DIRAC/Core/Tornado/Server/private/BaseRequestHandler.py

```diff
@@ -591,20 +591,22 @@ def on_finish(self):
         Log the request duration
         """
         elapsedTime = 1000.0 * self.request.request_time()
-
-        argsString = "OK"
-        try:
-            if not self.result["OK"]:
+        credentials = self.srv_getFormattedRemoteCredentials()
+
+        argsString = f"OK {self._status_code}"
+        # Finish with DIRAC result
+        if isinstance(self.result, dict) and isinstance(self.result.get("OK"), bool):
+            # S_OK:
+            if self.result["OK"] and "Value" in self.result:
+                argsString = "OK"
+            # S_ERROR:
+            elif self.result.get("Message"):
                 argsString = "ERROR: %s" % self.result["Message"]
```

Comment:
This could be `isReturnStructure` from `DIRAC.Core.Utilities.ReturnValues`

```suggestion
        if isReturnStructure(self.result):
            argsString = "OK" if self.result["OK"] else f"ERROR: {self.result['Message']}"
```